src/chart/SunburstChart.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { scaleLinear } from 'victory-vendor/d3-scale';
+import { Surface } from '../container/Surface';
+import { Layer } from '../container/Layer';
+import { Sector } from '../shape/Sector';
+import { Text } from '../component/Text';
+
+export interface SunburstData {
+  name: string;
+  value?: number;
+  fill?: string;
+  children?: SunburstData[];
+}
+
+interface TextOptions {
+  fontFamily?: string;
+  fontWeight?: string;
+  paintOrder?: string;
+  stroke?: string;
+  fill?: string;
+}
+
+export interface SunburstChartProps {
+  data?: SunburstData;
+  width?: number;
+  height?: number;
+  padding?: number;
+  ringPadding?: number;
+  innerRadius?: number;
+  children?: React.ReactNode;
+  fill?: string;
+  stroke?: string;
+  textOptions?: TextOptions;
+}
+
+interface DrawArcOptions {
+  r: number;
+  innerR: number;
+  initialAngle: number;
```

Comment:
Could you please add a comment, explaining in how far this angle is initial? Is it changing over time?

Reply:
I understand from reading the code - and wonder - should we also have an endAngle?